packages/calcite-components/src/components/time-picker/time-picker.tsx

```diff
@@ -202,7 +202,7 @@ export class TimePicker
   /**
    * @internal
    */
-  @Event({ cancelable: false }) calciteInternalTimePickerChange: EventEmitter<string>;
+  @Event({ cancelable: false }) calciteInternalTimePickerChange: EventEmitter<void>;
```

Comment:
never actually included a `string` payload type, and no one was listening for it